import "does_not_exist.ant"

x = failed_func()